Assets/Scripts/Utility/CameraClearManager.cs

```diff
@@ -53,12 +53,13 @@ void Update()
                 mainCamera.backgroundColor = cameraClearColor;
                 //Debug.Log("Camera clear set to inside");
 
+                // Comment this out, We're not Unity 5.5 anymore - MeteoricDragon
                 // Disable global fog inside
                 // This fixes an issue with solid camera clear not working in Unity 5.5
-                if (toggleGlobalFog && globalFog)
```

Comment:
I remember this issue - will need to review if problem still exists in Unity 2018.x and find another solution if so. No problems keeping commented for now.